__tests__/app.test.js

```diff
@@ -118,3 +117,50 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: Responds with an array of comments for the given article_id of which each comment should have the following properties:comment_id, votes, created_at, author, body, article_id. Comments should be served with the most recent comments first.", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body: { comments } }) => {
+        comments.forEach((comment) => {
```

Comment:
I'd recommend adding a check for the expected length of the comments array here too - might seem a weird suggestion at first but using assertions inside the forEach alone can pass given an empty array 😱